tests/backup/backup_helper.go

```diff
@@ -5434,3 +5434,128 @@ func RestartAllVMsInNamespace(namespace string, waitForCompletion bool) error {
 	}
 	return nil
 }
+
+// UpgradeKubevirt upgrades the kubevirt control plane to the given version
+// If workloadUpgrade is set to true, kubevirt workloads are upgraded to the given version
+func UpgradeKubevirt(versionToUpgrade string, workloadUpgrade bool) error {
+	k8sKubevirt := kubevirt.Instance()
+	k8sCore := core.Instance()
+
+	// Checking current version
+	current, err := k8sKubevirt.GetVersion()
+	if err != nil {
+		return err
+	}
+	log.Infof("Current version is - %s", current)
+
+	// Compare and validate the upgrade path
+	currentKubevirtVersionSemVer, err := version.NewSemver(strings.TrimSpace(strings.ReplaceAll(current, "v", "")))
+	if err != nil {
+		return err
+	}
+	versionToUpgradeSemVer, err := version.NewSemver(strings.TrimSpace(strings.ReplaceAll(versionToUpgrade, "v", "")))
+	if err != nil {
+		return err
+	}
+	if currentKubevirtVersionSemVer.GreaterThanOrEqual(versionToUpgradeSemVer) {
+		return fmt.Errorf("kubevirt cannot be upgraded from [%s] to [%s]", currentKubevirtVersionSemVer.String(), versionToUpgradeSemVer.String())
```

Comment:
Do we want to fail the test if its same version? Is the intenion to run tests after upgrade or is the test to upgrade kubevirt. If the test is to run upgrade and then execute test case.. you should not return error but return with log "already with upgraded version"